docs/dev/testing.rst

```diff
@@ -261,3 +262,45 @@ database containing all years of FERC 1 data (equivalent to ``tox -e ferc1_schem
 .. code-block:: console
 
   $ pytest --etl-settings test/settings/full-integration-test.yml test/integration/etl_test.py::test_ferc1_schema
+
+-------------------------------------------------------------------------------
+Data Validation
+-------------------------------------------------------------------------------
+Once the PUDL ETL pipeline has processed all of the available data, we have a
+collection of tests that can be run to verify a collection of expectations
+about what the outputs should look like. These data validation tests are
+organized into datasource specific modules under ``test/validate``. Running
+the full data validation can take as much as an hour, depending on your
+computer.
+
+These tests require a fully populated PUDL database which contains all
+available FERC and EIA data, as specified by the
+``test/settings/full-integration-test.yml`` input file. They are run against the
+"live" SQLite database in your pudl workspace at ``sqlite/pudl.sqlite``. To run
+the full data validation against an existing database:
+
+.. code-block:: console
+
+    $ tox -e validate
+
+Which internally calls pytest like this, but in a clean Python environment:
```

Comment:
I don't understand what this sentence means--specifically the latter part about a clean Python environment and why it wouldn't work in a "not clean" environment.

Reply:
Hmm. This is the major difference between pytest and Tox. Tox uses a separate Python environment to run the tests, which is created based on the `setup.py` file that's part of the packaging -- it determines how the software will get installed on a user's machine. So if you've installed some new package in your personal `conda-dev` environment without adding it to the dependencies that are automatically installed with PUDL is installed, the Tox tests should fail, but pytest might work fine, since it happens in your personal Python environment, whatever that happens to be when you call pytest.

Reply:
not sure if you removed this reference, but that explanation helped! Just didn't make sense to someone reading without that background knowledge.